src/main/java/com/autotune/analyzer/exceptions/KruizeErrorHandler.java

```
@@ -58,7 +58,9 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
                 .registerTypeAdapter(Date.class, new GsonUTCDateAdapter())
                 .create();
         String gsonStr = gsonObj.toJson(new KruizeResponse(origMessage, errorCode, "", "ERROR", myList));
-        LOGGER.error(gsonStr);
+        if (errorCode != HttpServletResponse.SC_CONFLICT) {
```

Please add a comment here about what is the error being suppressed here